_pages/backup-wallet/seeds-advanced.md

```diff
@@ -74,3 +74,32 @@ Place your paper into a plastic bag with a good seal (vacuum sealed even better)
 
 #### Fireproof Document Bag
 You can use these to protect seed backups, and also to protect your physical hardware wallets.
+
+## Example Backup Configuration
+
+| Wallet | 2 of 3                        | 3 of 5                        |
+|--------|-------------------------------|-------------------------------|
+| #1     | Safe at home                  | Safe at home                  |
+| #2     | Bank or trusted friend/family | Safe at home                  |
+| #3     | Bank or trusted friend/family | Bank or trusted friend/family |
+| #4     | -                             | Bank or trusted friend/family |
+| #5     | -                             | Bank or trusted friend/family |
+
+### Items to include with each wallet
+- With each wallet, include the following:
+  - the wallet's seed, on paper or metal
+  - all the wallets' master public keys[^1] (xpub), on paper
```

Comment:
I didn't know markdown supported footnotes, neat! I think it's probably still better to include the link/disclaimer inline though.